system/include/wasm_simd128.h

```diff
@@ -1,6 +1,16 @@
 /*
-WebAssembly SIMD128 Intrinsics
-*/
+ * Copyright 2016 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+/*
+ * WebAssembly SIMD128 Intrinsics
+ */
+
+#ifndef __WASM_SIMD128_H
+#define __WASM_SIMD128_H
```

Comment:
Actually the convention for emscripten headers is to use `#pragma once` instead.